recipes/cath-tools/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "cath-tools" %}
+{% set version = "0.16.5" %}
+{% set sha256 = "10b2913edf70cda698a47003d4f1ad8de89fc73ccacdafb1d75ce444be25bffe" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/UCLOrengoGroup/cath-tools/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('cxx') }}
+    - cmake >3.2
+    - gsl >=2.6
+    - llvm-openmp  # [osx]
+    - perl >=5.08
+    - pkgconfig
+    - pthread-stubs
+    - ripgrep
+    - libcxx
+  host:
+    - boost-cpp
+    - llvm-openmp  # [osx]
+    - cctools
+    - gsl >=2.6
+    - zlib
```

Comment:
Having these in the host environment will mean they'll be put there (and is why we see conda-build spending all that time going through files after compilation). Are they meant to be build or runtime requirements instead?

Reply:
Here what they says for installation:
https://cath-tools.readthedocs.io/en/latest/build/

I think it is not needed reading the CMakeLists.txt, everything (the libs) are installed in mode static, so it should not be needed at runtime but only to be build.